ansible_events_ui/api/project.py

```diff
@@ -1,3 +1,5 @@
+from http.client import ACCEPTED, CONFLICT, NOT_FOUND, UNPROCESSABLE_ENTITY
```

Comment:
Better to use `fastapi.status` for consistency, since the project is based on the FastAPI framework and it's already actively used in tests.